### PR TITLE
[UG] Update ug for some bugs and updates

### LINKED ISSUE #275: [PE-D][Tester A] Target users not specified in the user guide
From the name I can roughly figure that this is probably used for tutors (I am not sure private tutors or TAs) but I get the point. However, I believe that other users may not understand what this app is about and who it is for.

I would kindly suggest stating explicitly AT THE TOP the purpose of the app as well as its targeted end users.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: Al-ez/ped#8

### PR DESCRIPTION
**Pull request by NorbertLoh**
Fixes #275 #268 #260 and other updates

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -335,9 +335,9 @@ Priorities: MVP (must have), 2 (nice to have), 3 (unlikely to have)
...
+| Tutor | Add and delete my students' guardians' contacts                        | Keep track of my students’ guardians                     | MVP      |
 | Tutor | Keep track of the house address of a student for home tuition          | Easily go to their house when their tuition starts       | MVP      |
-| Tutor | Keep track of Zoom meeting link of a student for online tuition        | Easily go to the online meeting room when tuition starts | MVP      |
+| Tutor | Keep track of Zoom meeting link of a student for online tuition        | Easily go to the online meeting room when tuition starts | 2        |
 | Tutor | Store what was done in the lessons                                     | Track what was done and plan for next lessons easier     | 2        |
 | Tutor | Create and delete lesson slots in my schedule                          | Keep track of my lessons                                 | MVP      |
 | Tutor | List all my lesson slots in my schedule                                | Keep track of my lessons                                 | MVP      |
```
[the remark this answers is not in the document]
Comment (NorbertLoh) [reply docs/DeveloperGuide.md:343]: Ok good idea. I added it!

**Review by Chronoxy — APPROVED** [review]
LGTM 👍

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -354,11 +355,13 @@ Priorities: MVP (must have), 2 (nice to have), 3 (unlikely to have)
...
+| Tutor | Know if I have accidentally scheduled a class at a conflicting time slot | Avoid troubling students to reschedule after <br/>agreeing on a time slot | MVP      |
 | Tutor | Keep track of when and how much each student/guardian needs to pay       | Collect my fees timely and accurately                                     | 2        |
 | Tutor | Tag students under their guardian                                        | Track total fees to collect                                               | 2        |
 | Tutor | Automatically update the amount of fee I collect after a lesson          | Avoid manually update and track fees                                      | 2        |
 | Tutor | Batch delete all scheduled lessons with a student                        | Remove all students' classes                                              | 2        |
+| Tutor | Find a student's contact details quickly                                 | Contact them quickly                                                      | 2        |
```
Comment (ethantyh692) [inline docs/DeveloperGuide.md:363]: Actually is this not our find feature for both lessons and contacts?

**Review by ethantyh692 — APPROVED** [review]
LGTM

**Pull request merged**